src/Compilers/Core/Portable/MetadataReference/MetadataReference.cs

```diff
@@ -318,14 +332,10 @@ public static MetadataReference CreateFromAssembly(
             return CreateFromAssemblyInternal(assembly, properties, documentation);
         }
 
-        internal static PortableExecutableReference CreateFromAssemblyInternal(
+        internal static string GetAssemblyFilePath(
             Assembly assembly,
-            MetadataReferenceProperties properties,
-            DocumentationProvider? documentation = null)
+            MetadataReferenceProperties properties)
         {
-            // Note: returns MetadataReference and not PortableExecutableReference so that we can in future support assemblies that
```

Comment:
Should this comment be moved?

Reply:
No. This comment was originally on `CreateFromAssemblyInternal` and I changed that return type to be `MetadataImageReference`. The comment was incorrect before this change anyways because the return was `MetadataReference` not `PortableExecutableReferenceb